app/views/partials/foyer/capture-montant-ressource.html

```diff
@@ -29,14 +29,14 @@ <h3 class="panel-title">
         </span>
       </div>
       <div class="col-sm-3">
-        <label for="{{ index + ressource.type.id }}-annuel">
+        <label for="{{ ressource.type.id }}_annuel_{{ index }}">
           Total depuis {{ debutAnneeGlissante }}
         </label>
         <span class="input-group">
           <input
             type="number"
             class="form-control text-right"
-            id="{{ index + ressource.type.id }}-annuel"
+            id="{{ ressource.type.id }}_annuel_{{ index }}"
```

Comment:
Language and word inconsistency: `month` above, `annuel` here.

Maybe `{{ ressource.type.id }}-year_{{ index }}-month_{{ $index + 1 }}` and `{{ ressource.type.id }}-year_{{ index }}`?